src/instructlab/model/train.py

```diff
@@ -291,6 +291,12 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     config_sections=ADDITIONAL_ARGUMENTS,
     required=True,  # default from config
 )
+@click.option(
+    "--legacy",
+    is_flag=True,
+    help="if true, enables the legacy linux training code path from release 0.17.0 and prior.",
+)
```

Comment:
@RobotSail this was removed with @cdoern's training rewrite 
maybe you need to rebase?

Reply:
++ ah definitely